File: examples/breadcrumbs/breadcrumbs.php
<?php

// set breadcrumbs
$breadcrumbs = array();
$breadcrumbs[] = array('path' => '/', 'name' => 'Home');
$breadcrumbs[] = array('path' => '/kwartz-php',	'name' => 'Kwartz-php');
$breadcrumbs[] = array('path' => '/kwartz-php/examples/', 'name' => 'Examples');
$breadcrumbs[] = array('path' => '/kwartz-php/examples/breacrumbs', 'name' => 'Breadcrumbs');

// set title
$title = 'Example: Breadcrumbs';

// output
include('breadcrumbs.view');

?>

File: examples/breadcrumbs/breadcrumbs.rb
#!/usr/bin/ruby

## set breadcrumbs
breadcrumbs = [
   { :name => "HOME",        :path => "/", },
   { :name => "Kwartz",      :path => "/kwartz/", },
   { :name => "Examples",    :path => "/kwartz/examples/", },
   { :name => "Breadcrumbs", :path => "/kwartz/examples/breadcrumbs/", },
]

## set title
title = 'Result';

## display view
require 'erb'
filename = 'breadcrumbs.view'
str = File.open(filename) { |f| f.read }  ## or File.read(filename)
trim_mode = 2
ERB.new(str, $SAFE, trim_mode).run(binding())
